profiles/common/modules/custom/ecas/ecas_import_users/ecas_import_users.install

```diff
@@ -15,12 +16,13 @@ function ecas_import_users_uninstall() {
  * Implements hook_enable().
  */
 function ecas_import_users_enable() {
-  drupal_set_message('ECAS import users is now active on your site.');
+  $t = get_t();
```

Comment:
get_t should only run at installation phase.
Anyway, it does not harm, so it's not blocking.
https://api.drupal.org/api/drupal/includes%21bootstrap.inc/function/get_t/7.x

Reply:
Should we keep useless stuff in the code then?